org2jekyll.el

```diff
@@ -198,22 +208,22 @@ The `'%s`' will be replaced respectively by the blog entry name, the author, the
 (defalias 'org2jekyll/create-draft! 'org2jekyll-create-draft)
 
 (defun org2jekyll--list-dir (dir)
```

Comment:
We can use defalias then:
```lisp
(defalias 'org2jekyll--list-dir 'find-file)
```

or even delete it, the idea of having the function named otherwise is to have an explicit meaning